docs/docusaurus.config.js

```diff
@@ -135,7 +135,7 @@ module.exports = {
 				"docs": {
 					"routeBasePath": "/",
 					"sidebarPath": require.resolve("./sidebars.js"),
-					"editUrl": "https://github.com/dynamoose/dynamoose/edit/main/docs",
+					"editUrl": ({docPath}) => `https://github.com/dynamoose/dynamoose/edit/main/docs/docs_src/${docPath}`,
```

Comment:
I think this is more complicated than it needs to be. Couldn't we just do the following?

```suggestion
					"editUrl": "https://github.com/dynamoose/dynamoose/edit/main/docs/docs_src",
```